apollo-router/src/plugins/cache/entity.rs

```diff
@@ -1362,7 +1363,41 @@ fn extract_cache_keys(
 pub(crate) fn hash_entity_key(representation: &Value) -> String {
```

Comment:
Meta: the implementation changed but no unit tests changed. I think we need somthing that shows that two identical representations with different orderings product the same digest.

Reply:
I added unit test